packages/financial-templates-lib/src/price-feed/DefaultPriceFeedConfigs.js

```diff
@@ -280,6 +280,170 @@ const defaultConfigs = {
       ]
     }
   },
+  "[BTC-BASIS-3M]/USDC": {
+    type: "basis",
+    lookback: 7200,
+    minTimeBetweenUpdates: 60,
+    spotPriceFeeds: [
+      {
+        type: "medianizer",
+        computeMean: true,
+        medianizedFeeds: [{ type: "cryptowatch", exchange: "ftx", pair: "btcusd" }]
+      },
+      {
+        type: "medianizer",
+        computeMean: true,
+        medianizedFeeds: [{ type: "cryptowatch", exchange: "okex", pair: "btcusdc" }]
+      },
+      {
+        type: "medianizer",
```

Comment:
is there a reason why these all use a medianizer feed rather than just the cryptowatch feed directly?